apps/consumer-client/package.json

```diff
@@ -7,7 +7,9 @@
     "dev": "ts-node build.ts dev",
     "build": "tsc --noEmit && ts-node build.ts build",
     "lint": "tsc --noEmit && eslint \"**/*.ts{,x}\"",
-    "clean": "rm -rf node_modules public/js"
+    "clean": "rm -rf node_modules public/js public/artifacts",
+    "download:artifacts": "ts-node scripts/download-artifacts.ts",
+    "postinstall": "pcd-artifacts download"
```

Comment:
i'm pretty confused about this `pcd-artifacts download` command - I can't find anywhere that you're defining `pcd-artifacts`, what is that? did you mean for the `"postinstall"` script to be `"yarn download:artifacts"`?

Reply:
Yea, sorry for that. The package name is `artifacts`, and it was gitignored! I fixed it, and also removed that `download:artifacts` script there, it doesn't need it anymore.